### ISSUE TITLE
Manufacturer filter: unable to preselect multiple manufacturers

### ISSUE THREAD

**Issue by therampagerado** [thirtybees/blocklayered#20]
Similar to https://github.com/thirtybees/blocklayered/issues/17 if we like to preselect multiple manufacturers and use the prepared link somewhere (social media, direct link to client, showing a selection of products, or even on our mega menu to help the visitors navigate to certain group of products) this is not possible. Only the first record is kept in the URL.

Demo: https://test2.rampagesport.eu/en/coffee-and-tea#/manufacturer-bee_hive-test_manufacturer

**Comment by getdatakick** [comment]
I can't reproduce this issue on my local env. 

I've created another manufacturer, rebuild URL index, and then manually entered the URL.

`http://thirtybees.local/en/first-en#/manufacturer-another_manufacturer-bee_hive`

Worked properly - both manufacturers were selected. Worked even when I changed the order of manufacturers in URL

`http://thirtybees.local/en/first-en#/manufacturer-bee_hive-another_manufacturer`
